Gruntfile.js

```diff
@@ -979,12 +979,11 @@ module.exports = function(grunt) {
   ]);
 
   // CI tasks
-  grunt.registerTask('minify', 'Minify JS and CSS', [
-    'uglify:web',
+  grunt.registerTask('minify-admin', 'Minify JS and CSS', BUILD_NUMBER ? [
```

Comment:
Good idea adding the BUILD_NUMBER branch here so it's always called but is a no-op in dev. It's easy to miss though. Firstly it might be useful to create a constant boolean at the top of the file called `DEV`, eg: `const DEV = !BUILD_NUMBER;` - that way it better describes what it's actually checking. Secondly, the layout is a little hard to read (multi-line ternary) but I can't think of a better way to define it within the grunt task. What about...

```suggestion
  grunt.registerTask('minify-admin', 'Minify JS and CSS', DEV ? [] : [
     ...
  ]);
```